test/sjcl-ecdsa-signdeterministic-test.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+/*eslint-disable new-cap*/
+
+var assert = require('assert');
+var sjcl = require('ripple-lib').sjcl;
+
+describe.only('Deterministic ECDSA (RFC 6979)', function() {
```

Comment:
Left the `.only` in!  I downloaded the branch locally and indeed the 3 test cases, involving the rfc curve aren't working. 

Check another comment. The `c256` has (sadly) been monkey patched with `k256` as a deprecated backwards compat hack.